scalableminds/webknossos
# Allow version history view in annotations not owned by you

This PR enables the version history view for view only annotations / annotations where updates are not allowed. 

### URL of deployed dev instance (used for testing):
- https://allowversionhistoryinnotownedannotation.webknossos.xyz

### Steps to test:
- Create a new user (if not already existing) and activate that user.
- Log in as the new user and create a new annotation. Annotate around, whatever you like as long as it creates new versions of the annotation.
- Log out and log in again with the admin user. View the annotation of the newly created user. The `Restore Older Version` entry should now be available in the action bar dropdown menu. Prior to this pr, this item would not be enabled.
- Click this menu entry and play around with the version history view on the right. The elements in the version view should allow you to download the annotation of that version.

### Issues:
- fixes #6163

------
(Please delete unneeded items, merge only when none are left open)
